spiritual-outfitters.html

```diff
@@ -0,0 +1,212 @@
+---
+layout: default
+meta:
+  title: Spiritual Outfitters | Crossroads
+  description: Crossroads has a brand new look
+image:
+  url: "//crds-media.imgix.net/7EFusaW9y0WFSxO9KCdumH/2edc127ce50430b3894692f7075122fb/spiritual-outfitters.jpg"
+---
+
+{% include header.html %}
+<section class="bg-primary">
+    <div class="bg-video-player">
+        <div class="embed-responsive embed-responsive-custom">
+            <video width="1787" height="1005.1875" class="bg-jumbotron-video" autoplay="autoplay" loop="loop" muted=""
+                playsinline=""
+                src="//videos.ctfassets.net/y3a9myzsdjan/63BuO6LHW7hpVpx3mzEvp1/dde983bb80b2efabea608f8526d14f86/spiritual-outfitters-animation1.mp4">
```

Comment:
is there some other way to serve this video up instead of ctfassets?

Reply:
Not unless you want to put it on S3/Cloudfront.